source/e_2d_flood_tutorial.rst

```diff
@@ -0,0 +1,399 @@
+
+
+Drawing the dike (2D)
+^^^^^^^^^^^^^^^^^^^^^^
+Dikes are automatically read from the DEM. However, if the dike is narrow and the computation cells are large, it might be beneficial to draw the dike using a Linear Obstacle. Follow these steps, and reference the DEM and the :ref:`t4reference-image` and draw the obstacle:
+       
+#) In the *Layers* panel, locate the *2D* group, and select the *Linear Obstacle* layer.
+
+#) Enable editing mode by clicking the *Toggle editing mode* button (|toggle_editing|) located in the top left corner.
+
+#) Click on the *Add line feature* button (|add_line|).
+
+#) To begin drawing the dike, click on a starting location adjacent to the starting point of the channel, aligning it with the visible dike on the DEM. Next, trace the dike along the 'Scheepvaart- en Voedingskannal' visible in the DEM, and finally, select the location where you want the dike to end.
+
+#) Right-click to stop drawing the dike. A popup screen with the Feature Attributes should now appear.
+
+#) Fill in the value '3' for the *crest level* of the dike, then click *OK*.
+
+#)  Click the *Toggle editing mode* button in the toolbar to exit editing mode and save your edits to this layer.
+
+.. _adding_potential_breach:
+
+Potential Breach (1D-2D)
+^^^^^^^^^^^^^^^^^^^^^^^^
+Now that we have incorporated a connected channel into our model, the next step is to introduce a potential breach location. This breach acts as a link between the 1D and 2D aspects of the model. In our scenario, the potential breach will simulate a dike breach, allowing water from the channel to flow into the fields behind the dike. For more theoretical information on breaches, see: :ref:`breaches`. 
+
+See the :ref:`t4reference-image` for a reference where to draw the potential brach:
+
+#) In the *Layers* panel, in the *1D2D* group, click the *Potential breach* layer.
+
+#) Click the *Toggle editing mode* button (|toggle_editing|) in the top left.
+
+#) Click on the *Add line feature* button (|add_line|).
+
+#) Hover over your channel until a pink square appears. Click on the pink square. Now click on the other side of the dike. Right-click to stop drawing.
+
+#) Fill in the following parameters:
+
+   * ID: filled in automatically
+   * Code: a name to identify your potential breach
+   * Display name: a name to identify your potential breach
+   * Exchange level [m MSL]: 4
+   * Max breach depth [m]: 1
+   * Levee material: Sand
+   * Channel ID: filled in automatically
+
+#) Click *OK*.
+
+#) Click the *Toggle editing mode* button in the toolbar and save your edits to this layer.
+
+The 'Exchange level' represents the water level that the channel must reach to initiate a breach and exchange water. Additionally, a breach can also be "opened", similar to creating a hole in a dike. The 'Max breach depth' signifies the maximum depth of this opening, measured from the top of the dike.
```

Comment:
```suggestion
The 'Exchange level' represents the water level that the channel must reach to initiate a breach. Additionally, a breach can also be "opened", similar to creating a hole in a dike. The 'Max breach depth' signifies the maximum depth of the breach, measured from the top of the dike.
```
I don't understand what you mean by the opening of a breach (creating a hole in a dike)..

Reply:
als in dat een breach ook gemaakt kan worden zoals een gat in de dijk en het niet alleen maar een overstroming hoeft te zijn.